src/main/scala/com/microsoft/partnercatalyst/fortis/spark/Constants.scala

```diff
@@ -6,6 +6,8 @@ object Constants {
   val SparkStreamingBatchSizeDefault = 5
 
   val EventHubProgressDir = "eventhubProgress"
+  val ContextStopWaitTimeMillis = 5000
```

Comment:
Shouldn't the user need to click an `apply` button before we issue the restart command?